### ISSUE TITLE
MiKo_2040 should ignore normal code inside <code> tags

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#461]
It may happen that an XML comment contains a code example, such as following:
```c#
/// <code>
/// if (xyz is null)
///    return true;
/// </code>
```
Such code should NOT be reported as a violation.

On the other side, code such as following should be reported as a violation
```c#
/// <code>true</code>
```
